code/modules/projectiles/projectile/bullets/_incendiary.dm

```diff
@@ -8,10 +8,11 @@
 		var/mob/living/carbon/M = target
 		M.adjust_fire_stacks(fire_stacks)
 		M.IgniteMob()
-
+/*
 /obj/item/projectile/bullet/incendiary/Move()
```

Comment:
Don't comment out stuff just because you want things to work differently, particularly when the correct fix is to rearrange parents and hierarchies.